syntax-markdown-flavor/syntax-markdown-flavor-github/src/main/java/org/xwiki/contrib/rendering/markdown/flavor/github/internal/parser/AbstractMarkdownGithubStreamParser.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.contrib.rendering.markdown.flavor.github.internal.parser;
+
+import com.vladsch.flexmark.parser.Parser;
+import com.vladsch.flexmark.util.ast.Node;
+import com.vladsch.flexmark.util.data.MutableDataHolder;
+import org.apache.commons.io.IOUtils;
+import org.xwiki.contrib.rendering.markdown.markdown12.MarkdownConfiguration;
+import org.xwiki.contrib.rendering.markdown.markdown12.internal.parser.FlexmarkNodeVisitor;
+import org.xwiki.rendering.listener.Listener;
+import org.xwiki.rendering.parser.ParseException;
+import org.xwiki.rendering.parser.StreamParser;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.inject.Provider;
+import java.io.Reader;
+
+/**
+ * Base class for Markdown Streaming Parsers for the various Markdown flavors. Implemented using the
+ * <a href="https://github.com/vsch/flexmark-java">Flexmark Java Parser</a>.
+ *
+ * @version $Id$
+ * @since 8.4
+ */
+public abstract class AbstractMarkdownGithubStreamParser implements StreamParser
+{
+    @Inject
+    private Provider<FlexmarkNodeVisitor> visitorProvider;
+
+    @Inject
+    @Named("MarkdownConfigurationGithub")
+    private MarkdownConfiguration configuration;
+
+    @Override
+    public void parse(Reader source, Listener listener) throws ParseException
```

Comment:
Here again, this is duplicated from the default one. Basically what you'd need is a getConfiguration that could be overridden in all flavors.